src/PscIde/Command.purs

```diff
@@ -248,7 +263,8 @@ instance decodeTypeInfo :: DecodeJson TypeInfo where
     definedAt <- o `getFieldMaybe` "definedAt"
     expandedType <- o `getFieldMaybe` "expandedType"
     documentation <- o `getFieldMaybe` "documentation"
-    pure (TypeInfo { identifier, type', module', definedAt, expandedType, documentation })
+    exportedFrom <- Right $ either (const []) id $ getField o "exportedFrom"
```

Comment:
I've tried this against 0.11.4, 0.11.5, and the PR you just merged - the `either` is dealing with the case where it's the wrong type as well as simply absent property

(Maybe I should comment that?)